examples/platform/efr32/BUILD.gn

```diff
@@ -37,6 +37,11 @@ config("chip_examples_project_config") {
     "-Wl,--wrap=_free_r",
     "-Wl,--wrap=_calloc_r",
   ]
+
+  defines = [
+    # Set to 1 to enable EFR32 attestation credentials
+    "EFR32_ATTESTATION_CREDENTIALS=0",
+  ]
```

Comment:
Should be based on a GN args in  declare_args.

See https://github.com/project-chip/connectedhomeip/pull/20134/files#diff-9fa2fcb990b75b861b4192bcfb1b413098250be47ed5d1b4f7db23f0fd0bb4ac for an example of how to carry GN args down to defined, so that you don't have to edit BUILD.gn

Reply:
The new GN variable is `chip_build_device_attestation_credentials`